software/harvester-node/gpio.js

```diff
@@ -0,0 +1,31 @@
+const Gpio = require('onoff').Gpio;
+
+const buttons = {
+  1: new Gpio(6, 'in', 'both'),
+  2: new Gpio(24, 'in', 'both'),
+  3: new Gpio(12, 'in', 'both'),
+  4: new Gpio(13, 'in', 'both'),
+  5: new Gpio(22, 'in', 'both'),
+  6: new Gpio(17, 'in', 'both'),
+  7: new Gpio(16, 'in', 'both'),
```

Comment:
There seems to be an issue with pin 16 when I run this software. It always sees the resource as busby.  Comment this line out to ignore the issue for now.